orbit/models/dlt.py

```diff
@@ -38,10 +38,13 @@ class BaseDLT(BaseLGT):
     _model_name = 'dlt'
     _supported_estimator_types = None  # set for each model
 
-    def __init__(self, period=1, damped_factor=0.8, global_trend_option='linear', **kwargs):
+    def __init__(self, period=1, damped_factor=0.8, expected_size=None, global_shrinkage_factor=0.3,
```

Comment:
Can you explain `expected_size` a little more and also include that to docstrings